resources/js/features/game-list/components/HubMainRoot/HubBreadcrumbs/HubBreadcrumbs.tsx

```diff
@@ -0,0 +1,118 @@
+import { type FC, Fragment } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  BaseBreadcrumb,
+  BaseBreadcrumbItem,
+  BaseBreadcrumbLink,
+  BaseBreadcrumbList,
+  BaseBreadcrumbPage,
+  BaseBreadcrumbSeparator,
+} from '@/common/components/+vendor/BaseBreadcrumb';
+import { cleanHubTitle } from '@/common/utils/cleanHubTitle';
+
+interface HubBreadcrumbsProps {
+  breadcrumbs: Array<App.Platform.Data.GameSet>;
+}
+
+export const HubBreadcrumbs: FC<HubBreadcrumbsProps> = ({ breadcrumbs }) => {
+  const { cleanBreadcrumbHubTitles } = useCleanBreadcrumbHubTitles();
+
+  const seenPrefixes = new Set<string>();
+
+  return (
+    <div className="navpath mb-3 hidden sm:block">
+      <BaseBreadcrumb>
+        <BaseBreadcrumbList>
+          {breadcrumbs.map((breadcrumb, index) => {
+            const currentTitle = cleanBreadcrumbHubTitles(breadcrumb.title!, seenPrefixes);
+
+            return (
+              <Fragment key={`crumb-${breadcrumb.id}`}>
+                {index !== breadcrumbs.length - 1 ? (
+                  <>
+                    <BaseBreadcrumbItem aria-label={breadcrumb.title!}>
+                      <BaseBreadcrumbLink asChild>
+                        <a href={route('hub.show', { gameSet: breadcrumb.id })}>{currentTitle}</a>
+                      </BaseBreadcrumbLink>
+                    </BaseBreadcrumbItem>
+
+                    <BaseBreadcrumbSeparator />
+                  </>
+                ) : null}
+
+                {index === breadcrumbs.length - 1 ? (
+                  <BaseBreadcrumbItem aria-label={breadcrumb.title!}>
+                    <BaseBreadcrumbPage>{currentTitle}</BaseBreadcrumbPage>
+                  </BaseBreadcrumbItem>
+                ) : null}
+              </Fragment>
+            );
+          })}
+        </BaseBreadcrumbList>
+      </BaseBreadcrumb>
+    </div>
+  );
+};
+
+function useCleanBreadcrumbHubTitles() {
+  const { t } = useTranslation();
+
+  const cleanBreadcrumbHubTitles = (title: string, seenPrefixes: Set<string>): string => {
+    if (title === '[Central]') {
+      return t('All Hubs');
+    }
+
+    const cleaned = cleanHubTitle(title);
+
+    // Always strip these organizational prefixes first.
+    const alwaysStripPrefixes = [
+      'ASB -',
+      'Central - ',
+      'Events -',
+      'Genre - ',
+      'Meta - ',
+      'Meta|DevComp - ',
+      'Meta|QA - ',
```

Comment:
There's a `Meta|Art - ` prefix too:
![image](https://github.com/user-attachments/assets/ac3180e4-68d8-4fe7-9dd3-9647c3373810)

Reply:
Fixed in latest. I didn't notice this one because there's no parent for Meta|Art to roll up into (other than Central - Misc).